ex2/index.js

```diff
@@ -0,0 +1,12 @@
+function removeProperty(obj, property) {
+    const newObj = { ...obj };
+    delete newObj[property];
+   
+    return newObj;
+   }
```

Comment:
```suggestion
}
```
Se poate de lucrat la identare.
In general acoladele se inchid in aceeasi linie unde a fost deschisa, in cazul dat la linia 1. Daca acolo nu sunt spatii libere in fata, atunci nici aici nu se doreste